tests/core/test_collections.py

```diff
@@ -77,54 +74,3 @@ def test_manager():
     assert "loss" in cm.get("default").include_regex
     assert len(cm.get("default").tensor_names) > 0
     assert "total_loss" in cm.collections["trial2"].include_regex
-
-
-def test_collection_defaults_to_hook_config():
```

Comment:
why removed in the PR?

Reply:
moved to mxnet test directory (https://github.com/awslabs/sagemaker-debugger/pull/150#discussion_r389860200)